docs/reference/modules/indices/circuit_breaker.asciidoc

```diff
@@ -177,3 +177,64 @@ it triggers the regex circuit breaker and returns an error.
 +
 {es} only applies this limit if
 <<script-painless-regex-enabled,`script.painless.regex.enabled`>> is `limited`.
+
+[[eql-circuit-breaker]]
```

Comment:
This link is now the same as the one in the EQL documentation. Lets use a distinct/new one better to avoid clashes.

```suggestion
[[circuit-breakers-page-eql]]
```